micro-ui/web/micro-ui-internals/packages/modules/hrms/src/pages/createEmployee.js

```
@@ -142,8 +142,18 @@ const CreateEmployee = () => {
     history.replace(`/${window?.contextPath}/employee/hrms/response?isSuccess=true`, { id, message, showID: true, showChildren: false });
   };
 
+
+
   const onSubmit = async (data) => {
-    const hasCurrentAssignment = data?.Assignments?.some(assignment => assignment?.isCurrentAssignment === true);
+    const hasCurrentAssignment = data?.Assignments?.some(assignment => assignment?.isCurrentAssignment === true); 
+    let selectedCity= data?.Jurisdictions?.boundary;
```

Biome

[error] This let declares a variable that is only assigned once.

'selectedCity' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)